packages/suite/src/views/suite/SwitchDevice/SwitchDevice.tsx

```diff
@@ -41,6 +41,7 @@ export const SwitchDevice = ({ cancelable, onCancel }: ForegroundAppProps) => {
 
     return (
         <Modal
+            headingSize="large"
```

Comment:
I misguided you about this modal, sorry, it should be `default`, see https://www.figma.com/file/mQhdQPKzdi93Swb8FNX9wL/branch/pewjlOfequOKYVC6h3v0YL/%5Bs2.0%5D-Desktop?type=design&node-id=3276%3A171291&mode=dev

Reply:
:heavy_check_mark:

Reply:
Wallets is Title/Medium ;)

Reply:
this

<img width="498" alt="image" src="https://github.com/trezor/trezor-suite/assets/45338719/a6551439-985a-4552-ad11-ec48b9c086e7">

Reply:
So it seems to be correct, Wallets is set to `titleMedium` here, I am talking about the `SwitchDevice` modal above, but this is the `PassphraseModal`... it is quite confusing. But I believe it is correct now.